laravel/resources/views/profile.blade.php

```diff
@@ -117,5 +118,21 @@
         </form>
       </div>
     @endforeach
+    </div>
+    <br>
+    <p>新着</p>
+    <div>
+        @foreach ($user->follow_user_posts->take(6) as $post)
+        <br>
+        <p>フォロ－している
+            @if ($post->user->image ==  "no_image.png")
+                <p><img src="{{ asset('/SystemImage/no_image.png') }}" width="80px"></p>
+            @else
+                <p><img src="{{ asset("/PostImage/$post->user->image") }}" width="80px"></p>
+            @endif
+            {{ $post->user->name }}さんが新しい<a href="{{ route('comments.index',['post_id' => $post->id]) }}">{{ $post->title }}</a>を投稿しました！</p>
```

Comment:
こんな感じでも良いかなと思いましたが今のままでもOKです！
```suggestion
            {{ $post->user->name }}さんが「<a href="{{ route('comments.index',['post_id' => $post->id]) }}">{{ $post->title }}</a>」を投稿しました！</p>
```